pallets/passkey/src/lib.rs

```diff
@@ -137,7 +141,10 @@ pub mod module {
 				transaction_account_id.clone(),
 			));
 			let result = payload.passkey_call.call.dispatch(main_origin);
-			if result.is_ok() {
+			if let Ok(_inner) = result {
+				// all post-dispatch logic should be included in here
+				// let _ = PasskeyWeightCheck::<T>::post_dispatch(&payload, &inner); TODO: uncomment after fixing constraints
```

Comment:
There were some complications with generic constrains that I'm trying to figure out so that we can uncomment this and be able to execute `post_disptach` hooks

Reply:
related issue #2063

Reply:
Yep I am on the same boat ⛵